www/source/blog/2017-08-09-Habitat-plans-for-net-full-iis-apps.md

```diff
@@ -0,0 +1,301 @@
+[HAB-STUDIO] Habitat:\src> hab svc start mwrock/hab-sln
+hab-sup(MN): Supervisor starting mwrock/hab-sln. See the Supervisor output for more details.
+```
+
+Our supervisor log should show:
+
+```
+hab-sup(MR): Supervisor Member-ID 883fe61fe11b4e64b20245adc9830bed
+hab-sup(MR): Starting gossip-listener on 0.0.0.0:9638
+hab-sup(MR): Starting http-gateway on 0.0.0.0:9631
+hab-sup(MR): Starting mwrock/hab-sln
+hab-sln.default(SR): Hooks recompiled
+default(CF): Updated website.ps1 945a1ab52bb3ad41211cd77625dd870244be1a81b822e7db171795c4f22ade86
+hab-sln.default(SR): Configuration recompiled
+hab-sln.default(SR): Initializing
+hab-sln.default(SV): Starting service as user=matt, group=
+hab-sln.default(O): Ensuring xWebAdministration DSC resources are installed...
+hab-sln.default(O): Compiling DSC mof...
+hab-sln.default(O): WARNING: The names of some imported commands from the module
+hab-sln.default(O): 'PSDesiredStateConfiguration' include unapproved verbs that might make them less
+hab-sln.default(O): discoverable. To find the commands with unapproved verbs, run the Import-Module
+hab-sln.default(O): command again with the Verbose parameter. For a list of approved verbs, type
+hab-sln.default(O): Get-Verb.
+hab-sln.default(O): Applying DSC configuration...
+hab-sln.default(O):
+hab-sln.default(O): hab-sln is running
+```
+
+and browsing to `http://localhost:8099/hab_app` should bring up our ASP.NET web site. Now lets go crazy and change the web site port:
+
+```
+[HAB-STUDIO] Habitat:\src> 'port = 8098' | hab config apply hab-sln.default 1
+» Applying configuration for hab-sln.default incarnation 1
+Ω Creating service configuration
+✓ Verified this configuration is valid TOML
+↑ Applying to peer 127.0.0.1:9638
+★ Applied configuration
+```
+
+and our supervisor log reads:
+
+```
+hab-sln.default(SR): Hooks recompiled
+default(CF): Updated website.ps1 9d47f71105cd9971a516c3be4567dec3779f156cfaa72910304f3a79aca056f4
+hab-sln.default(SR): Configuration recompiled
+hab-sln.default(O): hab-sln is stoping...
+hab-sln.default(O): "hab_pool" successfully stopped
+hab-sln.default(O): "hab_site" successfully stopped
+hab-sln.default(O): hab-sln has stopped
+hab-sln.default(O): Ensuring xWebAdministration DSC resources are installed...
+hab-sln.default(O): Compiling DSC mof...
+hab-sln.default(O): WARNING: The names of some imported commands from the module
+hab-sln.default(O): 'PSDesiredStateConfiguration' include unapproved verbs that might make them less
+hab-sln.default(O): discoverable. To find the commands with unapproved verbs, run the Import-Module
+hab-sln.default(O): command again with the Verbose parameter. For a list of approved verbs, type
+hab-sln.default(O): Get-Verb.
+hab-sln.default(O): Applying DSC configuration...
+hab-sln.default(O):
+hab-sln.default(O): hab-sln is running
+```
+
+Our port change forces an application restart and now we can reach our application on `http://localhost:8098/hab_app`.
```

Comment:
Might be nice to show some screenshots of the app running on the different ports.